packages/sanity/src/core/hooks/useUserListWithPermissions.ts

```diff
@@ -72,14 +70,15 @@ export function useUserListWithPermissions(
 
   const [state, setState] = useState<UserListWithPermissionsHookValue>(INITIAL_STATE)
 
-  const list$ = useMemo(() => {
+  const [users$, systemGroup$] = useMemo(() => {
```

Comment:
Moved these to a separate memo, so the observables are stable across document changes.